Add `ASPNETCORE_ENVIRONMENT` variable (#6631)
umbraco/UmbracoDocs: 14/umbraco-cms/fundamentals/setup/server-setup/azure-web-apps.md

}
```

> Also remember to add environment variable `ASPNETCORE_ENVIRONMENT` witch value `Development`, `Staging` og `Production`.

Reviewer: A suggestion to rewrite a bit to improve readability - let me know what you think.

```suggestion
Remember to add an `ASPNETCORE_ENVIRONMENT` variable with values `Development`, `Staging`, or `Production`.
```
